apis/tempo/v1alpha1/microservices_webhook.go

```diff
@@ -98,6 +100,24 @@ func (d *Defaulter) Default(ctx context.Context, obj runtime.Object) error {
 		r.Spec.SearchSpec.DefaultResultLimit = &defaultDefaultResultLimit
 	}
 
+	// Default replicas for distibutor
+	if r.Spec.Components.Distributor == nil {
+		r.Spec.Components.Distributor = &TempoComponentSpec{}
+	}
+
+	if r.Spec.Components.Distributor.Replicas == nil {
+		r.Spec.Components.Distributor.Replicas = pointer.Int32(1)
```

Comment:
maybe we could define a var and reuse it for all repliacs

Reply:
> I think this should be covered with an E2E test

Wondering if we need to to an E2E test of this, as I'm not sure how to capture webhook errors on kuttl, is not enough to have a unit test for our webhook validator?